tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/MsTestV2EvpTests.cs

```diff
@@ -336,5 +362,116 @@ public async Task SubmitTraces(string packageVersion, string evpVersionToRemove,
                 throw;
             }
         }
+
+        [SkippableTheory]
+        [MemberData(nameof(GetDataForEarlyFlakeDetection))]
+        [Trait("Category", "EndToEnd")]
+        [Trait("Category", "TestIntegrations")]
+        [Trait("Category", "EarlyFlakeDetection")]
+        public async Task EarlyFlakeDetection(string packageVersion, string evpVersionToRemove, bool expectedGzip, string settingsJson, string testsJson, int expectedSpansForPre224, int expectedSpansForPost224)
```

Comment:
Well is an snapshot testing, it will fail if a tag is missing or the number of span changes... the EFD tests are mostly for count number, a more detailed tests is done in the normal integration test.